tests/MySphinxTestCase.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
You are right, should I put them directly under 'data' or create something like 'data/helpers'?